### ISSUE TITLE
Clone Notification is not working!!

### ISSUE THREAD

**Issue by karthikkonar24** [lordmilko/PrtgAPI#222]
I've tried to clone the notification trigger from one device to another. Using the below command.

 Get-Probe -Id XXXX | Get-Trigger -Type state -Inherited $false | Clone-Object -DestinationId XXXX

It's not throwing any error in Powershell. But cloning notification trigger is not happening. 

Correct Me!!! If I'm wrong.

**Comment by lordmilko** [comment]
Hi @karthikkonar24,

Are you able to advise what the output of

```powershell
Get-Probe -Id XXXX | Get-Trigger -Type state -Inherited $false

Get-Probe -Id XXXX | Get-Trigger -Type state -Inherited $false | Clone-Object -DestinationId XXXX -Verbose

Get-PrtgClient -Diagnostic
```

is?

Please obscure any sensitive details in the output from the second command such as your PRTG Server URL, username and passhash

Regards,
lordmilko

**Comment by karthikkonar24** [comment]
Hey lordmilko,
Please find the below response for your reference:
![image](https://user-images.githubusercontent.com/29901474/120103697-7e08c780-c16e-11eb-93c5-c0a250b970be.png)

**Comment by lordmilko** [comment]
Hi @karthikkonar24,

Based on the response to the first request, we can see that *there are no non-inherited state triggers* defined on the probe with ID 8334, hence why `Clone-Object` does nothing - there is nothing to clone!

If you have a look at the triggers defined on the probe with ID 8334 in the PRTG UI, are you able to advise whether any non-inherited state triggers display there?

Regards,
lordmilko

**Comment by karthikkonar24** [comment]
Hey lordmilko,

Device ID 8334 containing the notification trigger which I want to clone the same to the device id 9362. Please find the GUI view of the notification trigger which available on the ID 8334.
![ID-8334](https://user-images.githubusercontent.com/29901474/120114871-ccce5580-c19e-11eb-949b-b8c679bffd68.png)

**Comment by lordmilko** [comment]
Hi @karthikkonar24,

Are you able to advise what the output of

```powershell
Get-Probe -Id 8334
```

is?

**Comment by karthikkonar24** [comment]
Hi @lordmilko,

It's not returning any value after entering the command.
![image](https://user-images.githubusercontent.com/29901474/120125776-b4c6f800-c1d7-11eb-8863-53b3dfb51038.png)

But the 8334 is a device id, if do Get-device -id 8334 will return the value of device details.

**Comment by lordmilko** [comment]
Hi @karthikkonar24,

This appears to be a duplicate of #219

Can you please try and run the latest build of PrtgAPI via the [manual](https://github.com/lordmilko/PrtgAPI/wiki/Installation#manual) installation instructions and advise whether `Get-Probe -Id 8334` returns anything using that, and if so whether you can also retrieve and clone your triggers properly?

Regards,
lordmilko

**Comment by karthikkonar24** [comment]
Hi lordmilko,

As per your suggestions, I've downloaded the latest PRTGAPI but still Get-Probe -Id 8334 not returning any value. Any suggestion on it.

Regards,
Karthik Yadav

**Comment by lordmilko** [comment]
Hi @karthikkonar24,

Can you do

```powershell
Set-PrtgClient -LogLevel All

Get-Probe -Verbose

Get-Probe -Id 8334 -Verbose
```

and provide the output (please obscure any sensitive details like your server name, username and passhash int he results)

**Comment by lordmilko** [comment]
Hi @karthikkonar24,

Just following up on this. Another user also appeared to have this issue and it turned out they weren't in fact using the new build of PrtgAPI

To make sure you are using the new version of PrtgAPI, can you

1. Download the [latest build](https://ci.appveyor.com/api/projects/lordmilko/prtgapi/artifacts/PrtgAPI.zip)
2. Right click the downloaded **PrtgAPI.zip** file and go to Properties
3. On the *General* tab, under *Security* select **Unblock**
4. Unzip the file to `C:\temp\PrtgAPI`
5. Open a new PowerShell and type `ipmo C:\temp\PrtgAPI`
6. Provide the output of

```powershell
Get-Probe -Verbose
Get-Probe -Id 8334 -Verbose
(gmo prtgapi).path
```
Regards,
lordmilko

**Comment by karthikkonar24** [comment]
Hi @lordmilko, 
Please find the output. 
![PRTG1](https://user-images.githubusercontent.com/29901474/120454273-5c4b5280-c3b1-11eb-9430-bbba787777d0.png)
 

PS C:\Users\ABC> Set-PrtgClient -LogLevel All
PS C:\Users\ABC> Get-Probe -Verbose
VERBOSE: Get-Probe: Synchronously executing request
http://XXX-XXXX.COM/api/table.xml?content=probenode&columns=objid,name,condition,fold,groupnum,devicenum,up
sens,downsens,downacksens,partialdownsens,warnsens,pausedsens,unusualsens,undefinedsens,totalsens,schedule,basetype,bas
elink,notifiesx,intervalx,access,dependency,position,status,comments,priority,message,parentid,tags,type,active&count=*
&filter_parentid=0&username=XXX&passhash=XXXXXX
VERBOSE: Get-Probe: <probenode totalcount="0" listend="1">
  <prtg-version>20.3.61.1649</prtg-version>
</probenode>
PS C:\Users\ABC>
PS C:\Users\ABC> Get-Probe -Id 8334 -Verbose
VERBOSE: Get-Probe: Synchronously executing request
http://XXX-XXXX.COM/api/table.xml?content=probenode&columns=objid,name,condition,fold,groupnum,devicenum,up
sens,downsens,downacksens,partialdownsens,warnsens,pausedsens,unusualsens,undefinedsens,totalsens,schedule,basetype,bas
elink,notifiesx,intervalx,access,dependency,position,status,comments,priority,message,parentid,tags,type,active&count=*
&filter_objid=8334&filter_parentid=0&username=XXXX&passhash=XXXXXXXX
VERBOSE: Get-Probe: <probenode totalcount="0" listend="1">
  <prtg-version>20.3.61.1649</prtg-version>
</probenode>

**Comment by lordmilko** [comment]
Thanks @karthikkonar24,

Can you copy and paste the URL of the first request, i.e. this one

```
http://XXX-XXXX.COM/api/table.xml?content=probenode&columns=objid,name,condition,fold,groupnum,devicenum,up
sens,downsens,downacksens,partialdownsens,warnsens,pausedsens,unusualsens,undefinedsens,totalsens,schedule,basetype,bas
elink,notifiesx,intervalx,access,dependency,position,status,comments,priority,message,parentid,tags,type,active&count=*
&filter_parentid=0&username=XXX&passhash=XXXXXX
```

into the address bar of your web browser, hit enter, and then provide the contents of the `table.xml` file it downloads?

**Comment by lordmilko** [comment]
Hi @karthikkonar24,

I installed the same version of PRTG you're using (20.3.61.1649) and have managed to reproduce the issue. This issue is happening due to a bug that was present in this PRTG version that I actually opened a case with Paessler regarding at that time.

If you upgrade PRTG to version v20.3.62.1397 or higher you should find that `Get-Probe` is now functioning properly
